### ISSUE TITLE
Crnk client doesn't support bulk save operation

### ISSUE THREAD

**Issue by gratsias** [crnk-project/crnk-framework#817]
Checking the repository stub implementation, it seems that the bulk save is not supported yet.

https://github.com/crnk-project/crnk-framework/blob/1e33a2f2fb8d532fdfcb31929224856b795c2941/crnk-client/src/main/java/io/crnk/client/internal/ResourceRepositoryStubImpl.java#L47

**Comment by remmeier** [comment]
yes,.both on client and server. bulk post and delete is implemented, but not patch. it is implemented for operators/json patch, so it would not be far away if desired. has come up a few times with pr promises, but has not materialized as of yet.

I was in an immutable append-only world the past two years. for that reason not too much of a use for it, reflecting the current state on master.
